server/data/nl/danielsjf.txt

```diff
@@ -4029,3 +4029,190 @@ Ik heb net alle seizoenen van “Friends” in één ruk uitgekeken.
+Er vaart een ferry tussen Grand Canaria en Tenerife.
+Het is moeilijk om de herfstkleuren goed op een foto te krijgen.
+De oude man zat op een bankje te schilderen.
+Ik heb nog een schildersezel staan in de kelder.
+Je kan de code vinden op Github.
+De twee managers konden het niet goed met elkaar vinden.
+De muziek klonk luid op de achtergrond.
+Naar wat ben je aan het luisteren?
+Ik kan nog maar een handvol Chinese karakters lezen en schrijven.
+Ik volg momenteel een spoedcursus Kantonees.
+Mandarijn is echt Chinees voor mij.
+Ik heb nu een deelabbonement voor een elektrische step.
+Er zat een deur verstopt in de boekenkast.
+Toen hij klein was wou hij altijd uitvinder worden.
+Om op tijd te zijn moeten we ten laatste om vijf uur vertrekken.
+Ik heb een slimme stofzuiger van het merk Xiaomi.
+Om mijn lampen aan te sluiten gebruik ik het Zigbeeprotocol.
+Een concurrent van Zigbee is Z-Wave.
+Tegenwoordig heb je een app voor alles.
+De barometer geeft aan dat het weer gaat verslechteren.
+Om drie uur stond er nog steeds veel volk op de dansvloer.
+Bijt je soms op je nagels als je stress hebt?
+Zelfs na al de ontbering en uitputting, zag ze er nog steeds elegant uit.
+Haar vermoeidheid begint zijn tol te eisen.
+De lekkende kraan had ons veel geld gekost.
+Ken je het verschil tussen een citroen en een limoen?
+Bedoel je graden Celsius of Fahrenheit?
+Die vorm is niet aerodynamisch.
+Ik vind die anonimiteit wel prettig.
+Ik heb een anonieme brief verstuurd.
+Er missen nog wat vrouwelijke stemmen.
+Er is vooral nood aan extra drinkwater.
+Ik luister graag naar jazz.
+De vooruitgang is stilgevallen.
+We zijn vandaag naar een waterpretpark geweest. 
+Mijn neefje kan al in vier talen tot tien tellen.
+Ken jij alle landen van de wereld?
+Ik kan alle landen van de wereld aanduiden op een blinde kaart.
+De lamp flikkert maar helaas valt er niets aan te doen.
+De flikkerende lamp irriteert mij.
+Ben jij gedoopt?
+De muggenbeet onder mijn oksel jeukt al de hele dag.
+We hebben eten besteld bij de Mongool.
+Hij verkoopt altijd veel flauwekul.
+Met haar valt niet te discussiëren.
+De spion is erin geslaagd om jarenlang onder de radar te blijven.
+Haar ouders hebben een cruise langs de Bahama's gemaakt.
+Ik teken een driehoek op het bord.
+Een vierkant heeft vier gelijke zijden en vier rechte hoeken.
+Zet je de jacuzzi op zevenendertig graden?
+Ik twijfel nog of ik een bubbelbad ga installeren.
+De dynamo van mijn fiets was stuk.
+Gebruik jij je elektrische fiets vaak?
+De ritssluiting zit vast.
+Vergeet je duikbril en snorkel niet in te pakken.
+Zij heeft net haar diploma diepzeeduiken gehaald. 
+Oh lekker, garnalen zijn heerlijk.
+Ik heb haar een cadeau van Rituals gekocht.
+Dove is een bekend merk van Unilever.
+De dief had de kroonjuwelen gestolen.
+Voor het raam bevinden er zich tralies.
+Zijn poging om te ontsnappen was tevergeefs.
+Op welke datum vertrekken jullie?
+Op de betogers waren hun aanhangers talrijk aanwezig.
```

Comment:
"Op" is onlogisch, zou eerder "Bij" of omschrijven naar " de aanhangers van de betogers waren talrijk aanwezig"